setup.py

```diff
@@ -94,6 +93,7 @@
     'gammapy-iterative-source-detect = gammapy.scripts.iterative_source_detect:main',
     'gammapy-look-up-image = gammapy.scripts.look_up_image:main',
     'gammapy-model-image = gammapy.scripts.model_image:main',
+    'gammapy-make_bg_cube_models = gammapy.scripts.make_bg_cube_models:main',
```

Comment:
Please stick with the naming scheme to always use hyphens, not underscores in command line tools.
Hyphens are easier to type (no shift) and mixing hyphens and underscores will mean that users always have to guess.
`gammapy-make_bg_cube_models` -> `gammapy-make-bg-cube-models`